lib/classes/Variables.test.js

```diff
@@ -1359,6 +1360,55 @@ module.exports = {
         .populateProperty(property)
         .should.eventually.eql('my stage is dev');
     });
+
+    it('should support ${file(...)} syntax', () => {
+      return runServerless({
+        fixture: 'variables',
+        cliArgs: ['print'],
+      }).then(result => {
+        expect(result.serverless.service.custom.importedFile).to.deep.equal({
+          foo: 'bar',
+        });
+      });
+    });
+
+    it('should support ${file(...):key} syntax', () => {
+      return runServerless({
+        fixture: 'variables',
+        cliArgs: ['print'],
+      }).then(result => {
+        expect(result.serverless.service.custom.importedFileWithKey).to.equal('bar');
+      });
+    });
```

Comment:
As running `runServerless` instance is not light, it'll be more optimal if we minimize this calls. I think we can run just one `runServerless` in `before` and then in numerous `it`'s confirm on the outcome case by case.